### ISSUE TITLE
Remove duplicate profiles

### ISSUE THREAD

**Issue by McEileen** [lucyparsons/OpenOversight#583]
These two profiles appear to be duplicates.  (They have the same name and same first employment date.)
![seanaranasduplicate](https://user-images.githubusercontent.com/5529982/49766577-4a364080-fc9b-11e8-9493-c3dfed95943b.png)

Beside removing this specific duplicate, I wonder if there's any way to remove duplicates from the entire project?  It seems like it would require a mix of automation and human work.  

We could write a query to identify profiles that have the same department, first and last name, and first employment date?  Then a project member could manually check that they are actual duplicates before removing unnecessary profiles.  This would also require a step where information from the two profiles are combined.  Thoughts on this approach?

**Comment by sibrahi5** [comment]
Yep, pretty much catch suspected duplicates and then let humans make sure that they are indeed duplicates. If I can gain admin permissions for this department, I'd also delete this duplicate.

**Comment by camfassett** [comment]
Hey @sibrahi5 to clarify am I assigned here just to delete the duplicate, or is the ask to work on automation?

**Comment by ssempervirens** [comment]
Hey @McEileen do you still think writing this query makes sense? If no details have changed since this previous convo, I can take this on

**Comment by McEileen** [comment]
@ssempervirens Yes, we still need someone to work on this, thanks for checking in about it.  I think the previously outlined plan, where you write a query to catch suspected duplicates, then manually confirm they're duplicates before deleting them, makes sense.
